Cargo.toml

```diff
@@ -21,4 +21,4 @@ panic = 'unwind'
 [patch.crates-io]
 evm = { git = "https://github.com/purestake/evm", branch = "jeremy-optimize-disabled-tracing"}
 evm-runtime = { git = "https://github.com/purestake/evm", branch = "jeremy-optimize-disabled-tracing" }
-evm-gasometer = { git = "https://github.com/purestake/evm", branch = "jeremy-optimize-disabled-tracing" }
\ No newline at end of file
+evm-gasometer = { git = "https://github.com/purestake/evm", branch = "jeremy-optimize-disabled-tracing" }
```

Comment:
You might want to revert this ("no new line at end of file") to avoid merge conflicts